manticore/core/executor.py

```diff
@@ -16,6 +17,10 @@
 # This is the single global manager that will handle all shared memory among workers
 
 
+class ExecutorError(ManticoreError):
```

Comment:
It's not important. Raising is better than logging something.

Do we need it to be a ManticoreError at this level. Maybe ExecutorError is better. And let Manticore deal with executor exceptions?

Reply:
do you mean not have ExecutorError subclass ManticoreError? i think Manticore can still deal with executor exceptions in any case, but there is something i like about having this hierarchy.